Libraries/WebSocket/__mocks__/event-target-shim.js

```diff
@@ -6,4 +6,13 @@
 // problem go away.
 
 'use strict';
-module.exports = function() {};
+
+function EventTarget() {
+  // Support both EventTarget and EventTarget([list, of, events])
+  // as a super class, just like the original module does.
+  if (arguments.length === 1 && Array.isArray(arguments[0])) {
```

Comment:
Well, we never exercise that part of the code in the tests. The tests are pretty stupid. Like, really really stupid... :/